src/graphql/helpers.ts

```diff
@@ -23,6 +23,7 @@ export function formatSpace(id, settings) {
   space.voting.type = space.voting.type || null;
   space.voting.quorum = space.voting.quorum || null;
   space.voting.blind = space.voting.blind || null;
+  space.voting.hideAbstain = space.voting.hideAbstain || null;
```

Comment:
It should be default to false
```suggestion
  space.voting.hideAbstain = space.voting.hideAbstain || false;
```